server/authzhandler.go

```diff
@@ -22,6 +22,7 @@ import (
 	"go.kubeguard.dev/guard/authz"
 	"go.kubeguard.dev/guard/authz/providers/azure"
 
+	"github.com/golang/glog"
```

Comment:
We are only using `k8s.io/klog/v2` and avoid `glog`. So, please use that.

Reply:
removed glog